.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,40 @@
+<!--
+    Anything in <!-- and -> are comments and would not appear to us
+-->
+
+* [ ] I have read the [guidelines].
+
+<!--
+    We categorize our issues into 2 categories:
+    * Bug: Prefix your issue title with [BUG]
+    * Feature Request: Prefix your issue title with [FEATURE]
+    * Question: Prefix your issue title with [QUESTION]
+-->
+
+<!--
+    This section is only for Bug and Question issues
+-->
+* Sponge Implementation: <!-- SpongeForge or SpongeVanilla -->
+* Sponge build version: <!-- Find the version using /sponge version -->
+* Essence version: <!-- Find the version using /sponge plugins Essence -->
+* Plugins and Mods: <!-- Plugins and Mods installed on the server, use /sponge plugins to ensure to list all of them -->
+
+<!--
+    This section is only for Bug issues
+-->
+* Expected Result: <!-- What you expected -->
+* Current Result: <!-- What is the current situation -->
+* Steps to Reproduce: <!-- How can we replicate the problem -->
+* Server log: <!-- Optional, The server log, useful for finding -->
+* Media: <!-- Optional, Any images or videos to show the issue -->
+
+<!--
+    This section is only for Feature Request issues
+-->
+* API or Implementation: <!-- Is this for developers or players -->
```

Comment:
Uhhh... again, no. This should be something like "Feature requested". We can decide whether it is API or implementation.